app/models/aliases/miq_ems_operations_worker.rb

```diff
@@ -0,0 +1 @@
+::MiqEmsOperationsWorker = ManageIQ::Providers::BaseManager::OperationsWorker
```

Comment:
@agrare Should we remove this? aliases were only for backward compatibility, but this worker is new, so this is not needed (or do you need it because of the setttings?)

Reply:
Ah I thought these were used for shorthand, e.g. the refresh worker's alias is still used in MiqQueue.  I'll remove this then.